.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -85,7 +108,7 @@ jobs:
             vcpkg_triplet: x64-osx-release
             # FIXME: qmltestrunner fails to import QtQuick module
             # https://github.com/KDAB/cxx-qt/issues/110
-            ctest_args: --exclude-regex '^(cargo_clippy|cargo_doc|example_qml_features_test|example_qml_minimal_myobject_test|reuse_lint|cpp_clang_format|cargo_build_rerun|.*valgrind)$'
+            ctest_args: --exclude-regex '^(cargo_clippy|cargo_doc|example_qml_features_test|example_qml_minimal_myobject_test|cpp_clang_format|cargo_build_rerun|.*valgrind)$'
```

Comment:
`cpp_clang_format` can go as well.
Same for the other `ctest_args`

Reply:
done